app/controllers/microposts_controller.rb

```diff
@@ -60,10 +60,16 @@ def update
   def destroy
     respond_to do |format|
       if @micropost.destroy
-        format.html {redirect_to microposts_url, notice: t("microposts.destroyed")}
+        format.html do
+          redirect_to microposts_url,
+                      notice: t("microposts.destroy.success")
+        end
         format.json {head :no_content}
       else
-        format.html {redirect_to microposts_url, alert: t("microposts.destroy_failed")}
+        format.html do
+          redirect_to microposts_url,
+                      alert: t("microposts.destroy.failed")
```

Comment:
```suggestion
          redirect_to microposts_url,
                      alert: t(".failed")
```
Fix tương tự chỗ này